Corona-Warn-App/src/test/java/de/rki/coronawarnapp/risk/RiskLevelsTest.kt

```diff
@@ -1,16 +1,48 @@
 package de.rki.coronawarnapp.risk
 
 import com.google.android.gms.nearby.exposurenotification.ExposureSummary
+import de.rki.coronawarnapp.appconfig.AppConfigProvider
+import de.rki.coronawarnapp.nearby.ENFClient
 import de.rki.coronawarnapp.server.protocols.internal.AttenuationDurationOuterClass
+import io.kotest.matchers.shouldBe
+import io.mockk.MockKAnnotations
+import io.mockk.impl.annotations.MockK
 import junit.framework.TestCase.assertEquals
+import org.junit.Before
 import org.junit.Test
 
-class RiskLevelCalculationTest {
+class RiskLevelsTest {
```

Comment:
Please extend `BaseTest` which automatically set's up logging so that we can see log output if the classes produce any.